### PR TITLE
CB-1 Refactor and Fix Likes Api

### PR DESCRIPTION
**Pull request by onurcanavci**
Refactor likes ap according to best practice like seperating functions and fix some problems related with likes api

### CODE REVIEW

File: practice-app/backend/api/likes/likes.py [hunk 1]
```diff
@@ -1,48 +1,87 @@
...
 
-likes_bp = Blueprint('Likes of post', __name__)
+likes_bp = Blueprint('Like action and viewing user which likes post', __name__)
 
 @likes_bp.route('/api/post/<int:postId>/likes', methods=['GET'])
-
 def getLikes(postId):
+    postExist = findPostFromDb(postId)
+    userThatLikesPost = getLikesFromDb(postId)
+    numberOfPostLike = list(userThatLikesPost)
```
Comment (mrabiabrn) [inline practice-app/backend/api/likes/likes.py:11]: I could not see where you use this variable later in the function, it may be unnecessary

**Review by mrabiabrn — APPROVED** [review]
It seems to function as intended

**Review by erencanuysal — APPROVED** [review]
I can not see any problem

File: practice-app/backend/api/likes/likes.py (hunk 1 above)
Comment (onurcanavci) [reply practice-app/backend/api/likes/likes.py:11]: It was unnecessary, so I forgot to reply in here. Thanks

**Pull request merged**